api/app/signals/apps/email_integrations/tests/test_actions.py

```diff
@@ -172,6 +172,43 @@ def test_send_email_fails(self, mocked):
         self.assertFalse(self.action(signal, dry_run=False))
         self.assertEqual(len(mail.outbox), 0)
 
+    def test_allowed_contact_false(self):
+        """
+        Disable all emails when a signal has a feedback object with allows_contact False
+        """
+        status_text = FuzzyText(length=400)
+        signal = SignalFactory.create(status__state=self.state, status__text=status_text,
+                                      status__send_email=self.send_email, reporter__email='test@example.com')
+        feedback = FeedbackFactory.create(
+            _signal=signal,
+            allows_contact=False,
+            submitted_at='2022-01-01'
+        )
+        feedback.save()
+        signal.save()
+        self.assertFalse(self.action(signal, dry_run=False))
+        self.assertEqual(len(mail.outbox), 0)
+
+    @override_settings(FEATURE_FLAGS={
+        'REPORTER_MAIL_DISABLE_CONTACT_FEEDBACK_ALLOWS_CONTACT': False,
```

Comment:
See comment on `settings/base.py`. Rename feature flag to `REPORTER_MAIL_CONTACT_FEEDBACK_ALLOWS_CONTACT_ENABLED`